rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -433,6 +433,15 @@ error_code cellCameraInit()
 		g_camera.attr[CELL_CAMERA_AGCHIGH] = { 64 };
 		break;
 	}
+
+	case fake_camera_type::uvc1_1:
+	{
+		g_camera.attr[CELL_CAMERA_DEVICEID] = { 0x5ca, 0x18d0 };
```

Comment:
what ID is this? comment missing

Reply:
KBCR-S01MU
Seems games check for whatever model they used for a specific cabinet type, so may need to break this out to a config option in future

Reply:
Comment?